docs/sources/slack/discovery.yaml

```diff
@@ -11,6 +11,10 @@ endpoints:
       - !<pseudonymize>
         jsonPaths:
           - "$.users[*].id"
+        includeReversible: true
+        encoding: "URL_SAFE_TOKEN"
```

Comment:
Now *userId* from listing users needs to be url safe, as it is provided as as parameter for user conversation endpoint.